tests/pages/insights-owner.js

```diff
@@ -0,0 +1,32 @@
+import {
+  create,
+  collection,
+  text,
+  isVisible,
+  visitable
+} from 'ember-cli-page-object';
+
+export default create({
+  visit: visitable('/:username/?tab=insights'),
+
+  glances: collection('.insights-grid .insights-glance', {
+    name: text('.insights-glance__title'),
+    keyStat: text('.insights-glance__stat'),
+    percentChange: text('.insights-glance__change-stat > span'),
+
+    chartIsVisible: isVisible('.insights-glance__chart'),
+    chartPlaceholderIsVisible: isVisible('.insights-glance__chart-placeholder'),
+  }),
+
+  odysseys: collection('.insights-grid .insights-odyssey', {
+    name: text('.insights-odyssey__title'),
+    chart: text('.insights-odyssey__chart'),
+  }),
+
+  noBuildOverlay: {
+    scope: '.insights-overlay',
+    title: text('[data-test-insights-overlay-title]'),
+    text: text('[data-test-insights-overlay-text]'),
+    link: text('[data-test-insights-overlay-link]'),
+  }
+});

```

Comment:
I would suggest to use test selectors in tests, just to make it more flexible and easier to refactor. Test data attributes usually define the _meaning_ whereas CSS selectors define _appearance_, which is sometimes not the same and change differently during refactoring.

Also there's https://github.com/travis-ci/travis-web/blob/master/tests/helpers/selectors.js that could help to reuse selectors between integration and acceptance tests.

Reply:
Ahh cool yeah, that makes a lot of sense. I'm going to focus on the other PR for a while, but I've at least got all of the class selectors in this file switched over to test selectors